Mathlib/Init/Algebra/Classes.lean

```diff
@@ -8,56 +8,111 @@ Authors: Leonardo de Moura
 ! Please do not edit these lines, except to modify the commit id
 ! if you have ported upstream changes.
 -/
-prelude
-import Leanbin.Init.Logic
-import Leanbin.Init.Data.Ordering.Basic
-
-universe u v
+import Mathlib.Init.Logic
+import Mathlib.Init.Data.Ordering.Basic
 
 /-!
 # Unbundled algebra classes
 
-These classes and the `@[algebra]` attribute are part of an incomplete refactor described
+These classes are part of an incomplete refactor described
 [here on the github Wiki](https://github.com/leanprover/lean/wiki/Refactoring-structures#encoding-the-algebraic-hierarchy-1).
+However a subset of them are widely used in mathlib3,
+and it has been tricky to clean this up as this file was in core Lean 3.
 
-By themselves, these classes are not good replacements for the `monoid` / `group` etc structures
+By themselves, these classes are not good replacements for the `Monoid` / `Group` etc structures
 provided by mathlib, as they are not discoverable by `simp` unlike the current lemmas due to there
-being little to index on. The Wiki page linked above describes an algebraic normalizer, but it is not
```

Comment:
Are you removing this sentence on purpose?

Reply:
No, this is an accident

Reply:
Oh, actually: I was assuming that the removal was deliberate since the status of the implementation of a mathlib3 feature is irrelevant

Reply:
Hmm, I think I would just add "lean 3" or "mathlib 3" to the sentence, but keep it in place.